manifests/snapshot.pp

```diff
@@ -69,7 +69,7 @@
   file {
     "${script_path}/elasticsearch_backup.py":
       ensure => file,
-      source => 'puppet:///modules/elasticsearch/usr/local/bin/elasticsearch_backup.py',
+      source => "puppet:///modules/${module_name}/usr/local/bin/elasticsearch_backup.py",
```

Comment:
I could also put `oldelasticsearch` there literally, but it mentioned the `${module_name}` in other cases, so I thought to make it consistent.